docs/planning/backend/app/services/openrouter_client.py

```
@@ -0,0 +1,157 @@
+            "temperature": temperature,
+        }
+
+        with httpx.Client(timeout=60.0) as client:
+            response = client.post(
+                f"{self.BASE_URL}/chat/completions",
+                headers=self._get_headers(),
+                json=payload,
+            )
+
+            if response.status_code != 200:
+                raise Exception(f"OpenRouter API error: {response.status_code} - {response.text}")
```

Ruff (0.14.10)

Create your own exception

(TRY002)